tests/test_svg.py

```diff
@@ -0,0 +1,21 @@
+
+from dominate.tags import svg, circle, rectangle
+import unittest
```

Comment:
No need for `unittest` since we we're using pytest

Reply:
Switched to pytest.